### DISCUSSION TITLE
How to use A4 in Landscape mode?

### DISCUSSION THREAD

**Discussion by kittiu** [frappe/print_designer#163] [Q&A]
As in the question, I still not seeing this option. I am thinking the correct way is to use Page Size = CUSTOM, and define it by mm?

Even so, I think having landscape option that change the mm can be handy :+1:

**Comment by kittiu** [comment]
OK still got some other problem. I did try CUSTOM mode,
![Screenshot from 2024-02-13 11-49-42](https://github.com/frappe/print_designer/assets/1973598/75ee8284-6dad-4199-8df9-71c3043eb3c3)

But when I print it, it looks like this. It was squeezed into portrait. 
![Screenshot from 2024-02-13 11-52-32](https://github.com/frappe/print_designer/assets/1973598/a2f5b9e2-d36a-46ae-882d-7d7a9ab6106c)

But luckily, the PDF download still OK
![Screenshot from 2024-02-13 11-54-12](https://github.com/frappe/print_designer/assets/1973598/bfc9313c-e804-4d27-81dd-cc00185f6f4d)

**Reply by kholidraf** [reply]
You need to set "custom print format" as default for doctypes.
